src/rest/handlers/RequestHandler.js

```diff
@@ -68,34 +68,37 @@ class RequestHandler {
           resolve();
         }
       };
-      item.request.gen().end((err, res) => {
+      item.request.make().then(res => {
         if (res && res.headers) {
           if (res.headers['x-ratelimit-global']) this.manager.globallyRateLimited = true;
           this.limit = Number(res.headers['x-ratelimit-limit']);
```

Comment:
This should be

```javascript
this.limit = Number(res.headers.get('x-ratelimit-limit'));
```

In `node-fetch`, `res.headers` is not a plain object like in other libraries, but rather an instance of `Headers` which has a method of `.get`.